content/courses/onchain-development/local-setup.md

```diff
@@ -17,59 +18,85 @@ description:
 
 ## Lesson
 
-There's no lesson here! Let's install Solana CLI tools, the Rust SDK, and
-Anchor, and create a test program to ensure that our setup works.
+This lesson is a guide to installing the tools required for developing onchain
+programs. Let's install Solana CLI tools, the Rust SDK, and Anchor, and create a
+test program to ensure that our setup works.
 
 ## Lab
 
-#### Extra steps for Windows users
+### Extra steps for Windows users
 
-Firstly install
-[Windows Terminal](https://apps.microsoft.com/detail/9N0DX20HK701) from the
-Microsoft store.
+> macOS and Linux users can skip this section. Linux and macOS are the
+> recommended operating systems for Solana program development. However, if
+> you're on Windows, you can still follow along with these extra steps.
```

Comment:
I'm not completely comfortable with:

> Linux and macOS are the
> recommended operating systems for Solana program development

@nickfrosty can you please give a second opinion here?

Reply:
I thought it may be appropriate to add this, as on Windows you still need to get WSL for local setup, which is in a way setting up a Linux-like environment for it. Whereas, you can just get started much more quickly and easily.
Personally, I use Linux and Windows, and the experience is better on Linux hands down. 
And I've met a handful of devs in the ecosystem who prefers mac or linux

Reply:
Though leaving it to users could be better choice. 
Should I fix this part?

Reply:
We don't want Windows users to feel that they need to purchase an expensive Apple laptop or learn Linux just to program Solana. Just remove it. They already know these steps only apply to them.

Reply:
sure